src/Uno.Extensions.Templates/content/unoapp-extensions/.template.config/dotnetcli.host.json

```diff
@@ -37,9 +45,25 @@
       "longName": "windowsAppSdk",
       "shortName": "win"
     },
-    "tests": {
-      "longName": "tests",
-      "shortName": "tests"
+    "unittest": {
```

Comment:
Could we keep "tests" (default to true) but make it hidden - inclusion of say unit tests would be "tests && unittests"